src/Client.ts

```diff
@@ -271,7 +271,11 @@ export class Client extends EventEmitter {
 			case GatewayDispatchEvents.ChannelUpdate:
 				this.emit("channelUpdate", this._handleChannelPayload(payload));
 				break;
+			case GatewayDispatchEvents.ChannelDelete:
+				this.emit("channelDelete", this._handleChannelPayload(payload));
+				break;
 			default:
+				this.emit("unknown", payload);

```

Comment:
See the other comment.
```suggestion
				this.emit("unknownPayload", payload);
```